Classes/Command/QueueCommandController.php

```diff
@@ -59,6 +59,23 @@ public function listCommand()
     }
 
     /**
+     * Show details of a queue
+     *
+     * @param string $queue
+     * @return void
+     */
+    public function showCommand($queue)
```

Comment:
how about calling this command `describe` instead of show? I think show is rather ambiguous, as it also could mean "show me what's in there".